samples/rtl/axpy.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2019-2022 ETH Zurich and the DaCe authors. All rights reserved.
+#
+# This sample shows the AXPY BLAS routine. It is implemented through Xilinx IPs in order to utilize floating point
+# operations. It is intended for running hardware_emulation or hardware xilinx targets.
+
+import dace
+import numpy as np
+
+# add symbol
```

Comment:
I would have this as a `make_sdfg` function, where parameters (e.g., veclen) are passed as arguments

Reply:
This has been changed